eng/pipelines/coreclr/crossgen2.yml

```diff
@@ -57,6 +57,23 @@ extends:
             displayNameArgs: R2R_CG2
             liveLibrariesBuildConfig: Release
 
+      - template: /eng/pipelines/common/platform-matrix.yml
+        parameters:
+          jobTemplate: /eng/pipelines/common/global-build-job.yml
+          buildConfig: Release
+          platforms:
+            - linux_x64
+            - osx_x64
+            - windows_x64
+          jobParameters:
+            testGroup: innerloop
+            buildArgs: -s clr+libs+libs.tests
+                       -c $(_BuildConfig) -test
+                       /p:TestReadyToRun=true
+                       /p:RunSmokeTestsOnly=true
```

Comment:
`RunSmokeTestsOnly` will run exactly one test (System.Runtime.Tests) and nothing else. Each workload needs to define what the set of smoke tests looks like, or they only get System.Runtime.Tests. It almost doesn't seem worth to build the whole product just to run one test. Are there too many failures if RunSmokeTestsOnly is not specified? Or why do we restrict to smoke tests only? We typically run smoke tests on legs that run on all PRs by default. For on-demand legs we just run everything.

Reply:
I see, thanks for explaining this Michal. I set it to `-p:RunSmokeTestsOnly=true` because that's what the base templates I used to figure out how this works had. But I will remove it then in this case.